srcpkgs/libogdf/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'libogdf'
+pkgname=libogdf
+version=2018.03
+revision=1
+wrksrc=OGDF-snapshot
+build_style=cmake
+configure_args="-DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=/usr -DOGDF_EXTRA_CXX_FLAGS=-Wno-error=class-memaccess -DOGDF_EXTRA_CXX_FLAGS=-Wno-error=deprecated-declarations"
+make_build_args="-Wno-error=class-memaccess"
+hostmakedepends=""
```

Comment:
unzip should be in hostmakedepends= for the package to be extracted even when cross-compiling